src/controllers/trip.controller.js

```diff
@@ -216,10 +216,96 @@ class TripController {
    */
   static async getTripRequest(req, res) {
     const subjectID = req.params.tripRequestID;
-    const userID = req.user.id;
-    const data = await tripService.getTripRequest(subjectID, userID);
+    const data = await tripService.getTripRequest(subjectID);
     if (data) return response.successMessage(res, 'success', 200, data);
     return response.errorMessage(res, 'Trip not found', 404);
   }
+
```

Comment:
@mujohn26 This PR is introducing changes that are not relevant to its purpose. The PR contains codes related to updating trips and trip-requests yet this PR is supposed to contain codes that send notifications to a requester's manager. I think you branched out of edit-trip-request branch. Before you start off a new feature, always branch out from develop.